k8s_test_harness/util/k8s_util.py

```diff
@@ -172,7 +188,31 @@ def wait_for_daemonset(
             constants.K8S_DAEMONSET,
             name,
             "--timeout",
-            "60s",
+            "1s",
+        ]
+    )
+
+
+def wait_for_statefulset(
```

Comment:
we should probably just make this a `wait_for_resource` in the future. Let's leave it like this for now (as we would need to update all Rocks).